frontend/src/cards/MapCard.tsx

```diff
@@ -128,13 +128,6 @@ function MapCardWithKey(props: MapCardProps) {
 
         breakdownValues.sort.apply(breakdownValues, sortArgs);
 
-        if (
-          activeBreakdownFilter === "" ||
-          activeBreakdownFilter === undefined
-        ) {
-          setActiveBreakdownFilter(ALL || breakdownValues[0]);
-        }
-
```

Comment:
Is this intention?

Reply:
Yes, I don't know why this was written this way but it was generating a console warning and it isn't doing anything because it just always sets it to ALL, so I moved ALL to the default value in useState()